src/server/routes/add_label.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from "express";
+
+import { endWithStatusAndBody } from "../helpers/http";
+import { getBqConnection } from "../helpers/bigquery";
+
+type addLabelPayload = {
+  report_uuid?: string;
+  label?: string;
+};
+
+export default async function handleAddLabel(req: Request, res: Response) {
```

Comment:
I've changed mark_invalid to generic add_label, so it can be used for `valid` label (and possibly others in the future).